DQM/L1TMonitor/python/L1TStage2uGT_cff.py

```diff
@@ -28,7 +28,14 @@
 # sequences
 l1tStage2uGTOnlineDQMSeq = cms.Sequence(
     l1tStage2uGT +
-    # l1tStage2uGTTiming +
+#    l1tStage2uGTTiming +
```

Comment:
This can be removed, you can now compare with the other sequence to identify what's missing
```suggestion
```

Reply:
Thank you @perrotta! I've tested the suggested changes with a Run 3 input file and the timing plots are there as well.